upgrading/topics/keycloak/changes.adoc

```diff
@@ -226,7 +226,7 @@ logs in.
 
 ==== Non-standard token introspection endpoint removed
 
-In previous versions, Keycloak advertized two introspection endpoints: `token_introspection_endpoint` and `introspection_endpoint`.
+In previous versions, Keycloak advertised two introspection endpoints: `token_introspection_endpoint` and `introspection_endpoint`.
```

Comment:
I am not sure if we are explicitly using the EN UK or EN US

Reply:
@dreamcrash, thank you for your review. Even in US english I think it is advertise : see https://www.merriam-webster.com/dictionary/advertise, https://books.google.com/ngrams/graph?content=advertise%2Cadvertize&year_start=1800&year_end=2019&corpus=28&smoothing=3&case_insensitive=true#.

Reply:
Yep, you are correct.